src/app/test_executive/test_executive.ml

```diff
@@ -132,62 +132,65 @@ let report_test_errors ~log_error_set ~internal_error_set =
         Print.eprintf "\n" )
   in
   (* check invariants *)
-  if List.length log_errors.from_current_context > 0 then
-    failwith "all error logs should be contextualized by node id" ;
-  (* report log errors *)
-  Print.eprintf "\n" ;
-  ( match log_errors_severity with
-  | `None ->
-      ()
-  | `Soft ->
-      report_log_errors "Warning"
-  | `Hard ->
-      report_log_errors "Error" ) ;
-  (* report contextualized internal errors *)
-  color_eprintf Bash_colors.magenta "=== Test Results ===\n" ;
-  Error_accumulator.iter_contexts internal_errors ~f:(fun context errors ->
-      print_category_header
-        (max_severity_of_list (List.map errors ~f:fst))
-        "%s" context ;
-      List.iter errors ~f:(fun (severity, { occurrence_time; error }) ->
+  match log_errors.from_current_context with
+  | [] ->
```

Comment:
This match statement seems to be inverted now.

Reply:
Oops. My mistake indeed. Thanks!

Reply:
This should be fixed.